src/test/java/org/broadinstitute/barclay/help/DocumentationGenerationIntegrationTest.java

```diff
@@ -68,29 +78,147 @@
                 {HelpDoclet.class,
                         new File(inputResourcesDir + "helpTemplates/"),
                         new File(testResourcesDir + "help/expected/HelpDoclet"),
+                        indexFileName,
                         "html", // testIndexFileExtension
                         "html", // testOutputFileExtension
                         "html", // requestedIndexFileExtension
-                        "html"  // requestedOutputFileExtension
+                        "html", // requestedOutputFileExtension
+                        new String[] {}, // extraJavadocArgs
+                        false    // onlyTestIndex
                 },
                 // defaut doclet and templates using alternate index extension
                 {HelpDoclet.class,
                         new File(inputResourcesDir + "helpTemplates/"),
                         new File(testResourcesDir + "help/expected/HelpDoclet"),
+                        indexFileName,
                         "html",  // testIndexFileExtension
                         "html",  // testOutputFileExtension
                         "xhtml", // requestedIndexFileExtension
-                        "html"   // requestedOutputFileExtension
+                        "html",  // requestedOutputFileExtension
+                        new String[] {}, // extraJavadocArgs
+                        false     // onlyTestIndex
                 },
                 // custom doclet and templates
                 {TestDoclet.class,
                         new File(testResourcesDir + "help/templates/TestDoclet/"),
                         new File(testResourcesDir + "help/expected/TestDoclet"),
+                        indexFileName,
+                        "html", // testIndexFileExtension
+                        "html", // testOutputFileExtension
+                        "html", // requestedIndexFileExtension
+                        "html", // requestedOutputFileExtension
+                        new String[] {}, // extraJavadocArgs
+                        false    // onlyTestIndex
+                },
+                // custom bash doclet and templates
+                {BashTabCompletionDoclet.class,
```

Comment:
Optional if you don't have the time now, but I'd like to see the tab completion tests in a separate file. We'd have to share the common setup code somehow. If not now, I'd probably want to do it at some point.